styles.css

```diff
@@ -1 +1,10 @@
-
+#img-div {
```

Comment:
Please, kindly make use of a `class` instead of an `id`

`.img-div`

It becomes useful when you need to apply the same rulesets for different elements 😃